pkg/apis/policy/validation/validation.go

```diff
@@ -465,9 +480,47 @@ func ValidatePodSecurityPolicyUpdate(old *policy.PodSecurityPolicy, new *policy.
 	allErrs = append(allErrs, apivalidation.ValidateObjectMetaUpdate(&new.ObjectMeta, &old.ObjectMeta, field.NewPath("metadata"))...)
 	allErrs = append(allErrs, ValidatePodSecurityPolicySpecificAnnotations(new.Annotations, field.NewPath("metadata").Child("annotations"))...)
 	allErrs = append(allErrs, ValidatePodSecurityPolicySpec(&new.Spec, field.NewPath("spec"))...)
+	allErrs = append(allErrs, validatePodSecurityPolicySeccompAnnotationsSpecEqual(new.Annotations, &new.Spec)...)
 	return allErrs
 }
 
+func validatePodSecurityPolicySeccompAnnotationsSpecEqual(annotations map[string]string, spec *policy.PodSecurityPolicySpec) field.ErrorList {
```

Comment:
I think this approach is problematic on update, because an older client may attempt to modify the annotation without knowing that the field exists. I'm not sure what the recommended approach is.

Reply:
Hm, you're right. I'm not sure whether there's any other feature that's in such a limbo annotation-field state.

I wonder whether we could somehow solve this by duplicating the values between fields and annotations. It sounds like a very crazy idea as the validation might turn into a nightmare. Possibly we could allow modifications of only one of annotation/field at a time and then duplicate the values to the other one?